Src/FluentAssertions/Equivalency/IEquivalencyValidator.cs

```diff
@@ -2,6 +2,6 @@ namespace FluentAssertions.Equivalency
 {
     public interface IEquivalencyValidator
     {
-        void AssertEqualityUsing(IEquivalencyValidationContext context);
+        void RecursivelyAssertEquality(IEquivalencyValidationContext context);
```

Comment:
@dennisdoomen renaming this is a breaking change and should be reverted.

Reply:
How so?

Reply:
From the CoreFX [Breaking Change Rules]((https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#members))
>Removing or renaming a member, including a getter or setter from a property or enum members

Though I can't find any code on the internet that implements `IEquivalencyValidator` except `EquivalencyValidator`, [Every change breaks someone's workflow](https://xkcd.com/1172/) and last time we thought it didn't matter, it [turned out bad](https://github.com/fluentassertions/fluentassertions.json/issues/27).
I think the benefit of renaming a public method versus the risk of introducing a breaking change, does not seem justified.